Source/WebCore/editing/Editor.h

```diff
@@ -518,6 +518,7 @@ class Editor final : public CanMakeCheckedPtr<Editor> {
     WEBCORE_EXPORT void replaceSelectionWithFragment(DocumentFragment&, SelectReplacement, SmartReplace, MatchStyle, EditAction = EditAction::Insert, MailBlockquoteHandling = MailBlockquoteHandling::RespectBlockquote);
     WEBCORE_EXPORT void replaceSelectionWithText(const String&, SelectReplacement, SmartReplace, EditAction = EditAction::Insert);
     WEBCORE_EXPORT bool selectionStartHasMarkerFor(DocumentMarker::Type, int from, int length) const;
+    WEBCORE_EXPORT void selectionStartSetMarkerFor(DocumentMarker::Type, int from, int length, const String&);
```

Comment:
I wish the name could better reflect its purpose (for testing only)? Any ideas?

Reply:
ya I don’t have any ideas, I just copy and pasted the getter name essentially 😅

Reply:
Right, a `…ForTesting` prefix would help clarify that this method is (under normal circumstances) reachable from tests only. You could also use `RELEASE_ASSERT_NOT_REACHED_WITH_MESSAGE` instead of `RELEASE_ASSERT_NOT_REACHED` in that method, and then you don't need the comment anymore.